tests/examples/shims/beautiful.lua

```diff
@@ -34,6 +34,8 @@ local module = {
     border_color = "#6181FF"  ,
     border_width = 1.5        ,
 
+    prompt_bg_cursor = "#00FF7D",
+
```

Comment:
Huh? Got some details for what this commit fixes exactly? For example, an error message with traceback? How is this related with 8cc469322f8043b262b36dd7743bb330bdde753d?